### DISCUSSION TITLE
How to register a field without rendering as input

### DISCUSSION THREAD

**Discussion by khanakia** [ant-design/ant-design#28943] [General]
I want to register a field which will be hidden and i want set that field value only using form API.
So i do not want to render that field as HIDDEN input. But that field will be saved within the form object.

How can i achieve this ?

**Comment by nrayburn-tech** [comment]
Look at the “noStyle” attribute on the “Form.Item” component. It should do what you need. I don’t think this renders anything in the dom.
Another alternative is to just use “hidden=true” on the input, or style=“display: none”. This renders in the dom, but is visually hidden.
